test/mason/publish-tests/noFork.chpl

```diff
@@ -5,7 +5,7 @@ use MasonUtils;
 
 proc badDryRun() throws {
   try! {
-    dryRun('this user name has not forked the repo');
+    dryRun('dsklnfdsklafndsklanf');
```

Comment:
I might use a self-documenting dummy value like `DummyGitUserName`.

Reply:
I blame sam for suggesting that exact string ;) @Spartee